### PR TITLE
Update guest home page

### PR DESCRIPTION
**Pull request by Khalid-Abed**

### CODE REVIEW

File: resources/views/guest/mainPage.blade.php [hunk 1]
```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: Change title to 'Land a job'

File: resources/views/guest/mainPage.blade.php [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+        </div>
+    </div>
+
+    <div class="container mt-5">
+        {{-- Latest Job   --}}
+        <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Latest Job</h1>
+       @if ($jobs->count() > 0)
+            <div class="row">
+                @foreach ($jobs as $index => $job)
+                    @if($index < 8)
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: Get only 8 latest records from ```database``` and remove this condition

File: resources/views/guest/mainPage.blade.php [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+                                </div>
+                            </div>
+                        </div>
+                    @endif
+                @endforeach
+            <div class="col-sm-12">
+                <div class="float-right my-3">
+                    <a href="#" >See all new Jobs on Land A Job</a>
+                </div>
+            </div>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php:81]: If this is possible and there are no jobs.
don't show this section only, don't tell the user.

File: resources/views/guest/mainPage.blade.php [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+        </div>
+       @else
+        <div class="container text-center">
+            <h1>There is no jobs Yet</h1>
+        </div>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: If this is possible and there are no jobs.
Don't show this section only, don't tell the user.

File: resources/views/guest/mainPage.blade.php [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+                            <h5 class="position-absolute text-light text-bottom p-2">{{$careerlevel->name}}</h5>
+                        </div>
+                    </div>
+                @endif
+            @endforeach
+        </div>
+      @else
+            <div class="container text-center">
+                <h1>There is no career level Yet</h1>
+            </div>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: If this is possible and there are no career levels.
Don't show this section only, don't tell the user.

File: resources/views/guest/mainPage.blade.php [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+        {{--  Jobs By Career Level  --}}
+        <h1 class="mb-3 mt-5 font-weight-bold text-secondary">Jobs By Career Level</h1>
+      @if ($careerlevels->count() > 0)
+        <div class="row ">
+            @foreach ($careerlevels as $index => $careerlevel)
+                @if($index < 6)
+                    <div class="col-lg-4 col-sm-12 my-3">
+                        <div class="box-shad">
+                            <a href="#">
+                                <img class="career-level" src="{{asset('img/guest/career/'.$career[$index])}}" alt="Not Found">
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]:
```html
<img alt = "Not Found>
```
```alt``` use to display words to tell users information to know what is the image.
Write instead 
```html
<img alt = "Career level">
```

File: resources/views/guest/mainPage.blade.php [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+        <div class="jobs-by-category my-5">
+            <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Browse Jobs by Category</h1>
+            @if ($industries->count() > 0)
+                <div class="row ">
+                    @foreach ($industries as $index => $industry)
+                        @if($index < 3)
+                            <div class="col-lg-4 col-sm-12 my-4">
+                                <div class="box-shad">
+                                    <a href="#">
+                                        <img class="career-level" src="{{asset('img/guest/category/'.$category[$index])}}" alt="Not Found">
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]:
```html
<img alt = "Not Found>
```
```alt``` use to display words to tell users information to know what is the image.
Write instead 
```html
<img alt = "industry category">
```

File: resources/views/guest/mainPage.blade.php [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+        <div class="jobs-by-category my-5">
+            <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Browse Jobs by Location</h1>
+           @if ($cities->count() > 0)
+                <div class="row ">
+                    @foreach ($cities as $index => $cit)
+                        @if($index < 3)
+                            <div class="col-lg-4 col-sm-12 my-4">
+                                <div class="box-shad">
+                                    <a href="#">
+                                        <img class="career-level" src="{{asset('img/guest/city/'.$city[$index])}}" alt="Not Found">
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]:
```html
<img alt = "Not Found>
```
```alt``` use to display words to tell users information to know what is the image.
Write instead 
```html
<img alt = "City">
```

File: resources/views/guest/mainPage.blade.php [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+        </div>
+       @endif
+
+        <hr class="my-4 mx-5">
+        {{--  Jobs By Career Level  --}}
+        <h1 class="mb-3 mt-5 font-weight-bold text-secondary">Jobs By Career Level</h1>
+      @if ($careerlevels->count() > 0)
+        <div class="row ">
+            @foreach ($careerlevels as $index => $careerlevel)
+                @if($index < 6)
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php:95]: Get only 6 records from ```database``` and remove this condition

File: resources/views/guest/mainPage.blade.php [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+      @endif
+
+        <hr class="my-4 mx-5">
+        {{--  Browse Jobs by Category --}}
+        <div class="jobs-by-category my-5">
+            <h1 class="mt-5 mb-3 font-weight-bold text-secondary">Browse Jobs by Category</h1>
+            @if ($industries->count() > 0)
+                <div class="row ">
+                    @foreach ($industries as $index => $industry)
+                        @if($index < 3)
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php:120]: Get only 3 records from ```database``` and remove this condition

File: resources/views/guest/mainPage.blade.php [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+                            <div class="col-lg-4 col-sm-12 my-4">
+                                <div class="box-shad">
+                                    <a href="#">
+                                        <img class="career-level" src="{{asset('img/guest/category/'.$category[$index])}}" alt="Not Found">
+                                    </a>
+                                    <h5 class="position-absolute text-light text-bottom p-2">{{$industry->name}} Jobs</h5>
+                                </div>
+                            </div>
+                        @elseif($index < 15)
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: Get only 15 records from ```database``` and remove this condition

File: resources/views/guest/mainPage.blade.php [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+                    <button class=" btn btn-primary  w-100 d-lg-none justify-content-center d-flex" type="button" data-toggle="collapse" data-target="#city-jobs" aria-controls="city-jobs" aria-expanded="true" aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                        show more
+                    </button>
+                </div>
+                </div>
+            @else
+                <div class="container text-center">
+                    <h1>There is no cities Yet</h1>
+                </div>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/guest/mainPage.blade.php]: If this is possible and there are no cities.
Don't show this section only, don't tell the user.

File: resources/views/guest/mainPage.blade.php (hunk 1 above)
Comment (Khalid-Abed) [reply resources/views/guest/mainPage.blade.php]: done

File: resources/views/guest/mainPage.blade.php (hunk 2 above)
Comment (Khalid-Abed) [reply resources/views/guest/mainPage.blade.php]: done

File: resources/views/guest/mainPage.blade.php (hunk 4 above)
Comment (Khalid-Abed) [reply resources/views/guest/mainPage.blade.php]: done

**Review by ahmed-fawzy-metwally — APPROVED** [review]

**Review by Ahmed-Elhadary — APPROVED** [review]

**Pull request merged**